apps/grader-host/src/index.ts

```diff
@@ -397,7 +406,7 @@ async function runJob(context: Context, receivedTime: Date, tempDir: string) {
           },
         ],
       },
-      Entrypoint: entrypoint.split(' '),
+      Entrypoint: shlex.split(entrypoint),
```

Comment:
If someone is able to test the grader host updated functionality, that would be great. This can be done with the new alpine question in testCourse (`externalGrade/entrypointArgs`), which has an entrypoint with quoted arguments and a grade.py file that is purposefully not executable in the repo.